Troubleshooting.md

```diff
@@ -0,0 +1,39 @@
+# Troubleshooting
+
+Here are some of the most common issues you might encounter when using a Skywallet, and the best possible resolutions for troubleshooting these issues. 
+
+If you do not find a resolution for your issue, make sure you contact the support team via - **[store.skycoin.net/pages/support](store.skycoin.net/pages/support)**.
+
+You can also send an email at **support@skycoin.zendesk.com**.
+
+## 1. Lost/Stolen Skywallet
+
+If your Skywallet is lost or stolen, the recommended solution is to **get a new one** as soon as possible. 
+
+Once you got the new Skywallet, then restore the Coins using the Seed of your old Skywallet.
+
+Even though **not a recommended** method, as an emergency step, you can also create a new Desktop Wallet to restore the lost Skywallet.
+
+This method is unrecommended and should only be opted as a last resort, as entering the Seed of your Skywallet into any electronic device could potentially compromise the security of the Seed of your Skywallet and hence putting your Coins at risk.
```

Comment:
This method is not recommended and should only be used if you cannot get a new Skywallet, ...

Reply:
Also not sure why we are capitalizing `Coins`